### PR TITLE
implement tutor main info block

### PR DESCRIPTION
**Pull request by Tolik170**
image

### CODE REVIEW

File: src/components/app-chips-list/AppChipList.js [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
   const hideChips = items.length - defaultQuantity > 0 && items.length - defaultQuantity
 
-  const chips = itemsForChips.map((item) => (
-    <AppChip handleDelete={ () => handleChipDelete(item) } key={ item }>
-      { item }
-    </AppChip>
-  ))
+  const chips = items.map((item) =>
+    handleChipDelete ? (
+      <AppChip handleDelete={ () => handleChipDelete(item) } key={ item }>
```
Comment (MickhaelSh) [inline src/components/app-chips-list/AppChipList.js:14]: move this to props const, const props = handleChipDelete ? { handleDelete :  () => handleChipDelete(item) } : { icon }
<AppChip {...props} key={ item }>

Comment (MickhaelSh) [inline src/components/app-chips-list/AppChipList.js:14]: <AppChip {...props} key={ item }>

const props = handleChipDelete ? { handleDelete : () => handleChipDelete(item) } : { icon }

File: src/components/app-popover/AppPopover.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <Box ref={ anchorEl }>
+      <Box sx={ { ...hideElement, ...initialItemsWrapperStyle } }>
+        { initialItems }
+        <Box onClick={ openPopover } sx={ { display: 'inline-block' } }>
+          { showMoreElem }
+        </Box>
+      </Box>
+
+      <Popover
+        anchorEl={ anchorEl.current } data-testid={ testId } onClose={ closePopover }
```
Comment (MickhaelSh) [inline src/components/app-popover/AppPopover.js:30]: data-testid={ isOpen && 'app-popover'
variable is no needed for this

File: src/components/icon-with-text-list/IconWithTextList.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  </Box>)
+
+  return (
+    <AppPopover
+      PaperProps={ { sx: styles.paperProps } }
+      TransitionProps={ { timeout: 500 } }
+      anchorOrigin={ styles.anchorOrigin }
+      hideElem
+      initialItems={ initialItems }
+      showMoreElem={ !shouldShowMore && showMoreElem }
```
Comment (MickhaelSh) [inline src/components/icon-with-text-list/IconWithTextList.js:46]: should not show more but u pass an element, maybe shouldShowMore && showMoreElem will be more logical?

Comment (MickhaelSh) [reply src/components/icon-with-text-list/IconWithTextList.js:46]: and u can move this check to const with element from the props

File: src/constants/translations/en/tutor-profile-page.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (MickhaelSh) [inline src/constants/translations/en/tutor-profile-page.json:1]: add Ukrainian translation too

File: src/containers/tutor-profile/main-info/MainInfo.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    navigator.clipboard.writeText(window.location.href)
+    setAlert({
+      severity: snackbarVariants.success,
+      message: 'tutorProfilePage.mainInfo.copyProfileLink',
+      duration: 2000
+    })
+  }
+
+  const editOrCopyIcon =
+    userRole === 'student' ? (
```
Comment (MickhaelSh) [inline src/containers/tutor-profile/main-info/MainInfo.js:41]: if its another tutor? you shold check is this my profile or not

File: src/containers/tutor-profile/main-info/MainInfo.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+        { isMobile && nameAndIcon }
+      </Box>
+
+      <Box sx={ styles.infoWrapper }>
+        { !isMobile && nameAndIcon }
+
+        <Box sx={ styles.chipsWrapper }>
+          <AppChipList
+            defaultQuantity={ 2 }
+            icon={ <SchoolIcon fontSize='small' sx={ { pl: '5px', fill: '#607D8B' } } /> }
```
Comment (MickhaelSh) [inline src/containers/tutor-profile/main-info/MainInfo.js:96]: what is that color?

File: src/pages/tutor-profile/TutorProfile.js [hunk 7]
```diff
@@ -1,5 +1,13 @@
+import Box from '@mui/material/Box'
+import MainInfo from '~/containers/tutor-profile/main-info/MainInfo'
+
 const TutorProfile = () => {
-  return <div>TutorProfile Page Placeholder</div>
+  return (
+    <Box sx={ { maxWidth: '1158px', margin: '0 auto' } }>
```
Comment (MickhaelSh) [inline src/pages/tutor-profile/TutorProfile.js:6]: use <Container> to wrap page and give flex 1 to it

File: src/styles/app-theme/app.typography.js [hunk 8]
```diff
@@ -45,6 +45,12 @@ const appTypography = {
     lineHeight: '20px',
     letterSpacing: '0.1px'
   },
+  subtitle3: {
```
Comment (MickhaelSh) [inline src/styles/app-theme/app.typography.js:48]: it looks almost the same as 'button' do we really need it?

File: src/components/app-chips-list/AppChipList.js (hunk 1 above)
Comment (Tolik170) [reply src/components/app-chips-list/AppChipList.js]: fixed

File: src/components/app-popover/AppPopover.js (hunk 2 above)
Comment (Tolik170) [reply src/components/app-popover/AppPopover.js]: fixed

File: src/components/icon-with-text-list/IconWithTextList.js (hunk 3 above)
Comment (Tolik170) [reply src/components/icon-with-text-list/IconWithTextList.js]: fixed, but i didn`t move it to const, for me it looks better

File: src/constants/translations/en/tutor-profile-page.json (hunk 4 above)
Comment (Tolik170) [reply src/constants/translations/en/tutor-profile-page.json:1]: fixed

File: src/containers/tutor-profile/main-info/MainInfo.js (hunk 5 above)
Comment (Tolik170) [reply src/containers/tutor-profile/main-info/MainInfo.js]: fixed

File: src/containers/tutor-profile/main-info/MainInfo.js (hunk 6 above)
Comment (Tolik170) [reply src/containers/tutor-profile/main-info/MainInfo.js]: fixed

File: src/styles/app-theme/app.typography.js (hunk 8 above)
Comment (Tolik170) [reply src/styles/app-theme/app.typography.js]: fixed

File: src/pages/tutor-profile/TutorProfile.js (hunk 7 above)
Comment (Tolik170) [reply src/pages/tutor-profile/TutorProfile.js]: fixed

File: src/containers/tutor-profile/main-info/MainInfo.jsx [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+import { styles } from '~/containers/tutor-profile/main-info/MainInfo.styles'
+import { snackbarVariants } from '~/constants'
+
+const MainInfo = () => {
+  const { t } = useTranslation()
+  const { isDesktop, isMobile } = useBreakpoints()
+  const { pathname } = useLocation()
+  const { setAlert } = useContext(SnackBarContext)
+
+  const isMyProfile = pathname.includes('/myProfile')
```
Comment (MickhaelSh) [inline src/containers/tutor-profile/main-info/MainInfo.jsx:31]: check this with router and use constant for path

Comment (Tolik170) [reply src/containers/tutor-profile/main-info/MainInfo.jsx]: fixed

**Review by MickhaelSh — APPROVED** [review]

File: src/containers/tutor-profile/main-info/MainInfo.jsx [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+import IconWithTextList from '~/components/icon-with-text-list/IconWithTextList'
+
+import useBreakpoints from '~/hooks/use-breakpoints'
+import { SnackBarContext } from '~/context/snackbar-context'
+import img from '~/assets/img/tutor-profile-page/avatar.png'
+import { accountInfo, subjectChips, doneItems } from '~/containers/tutor-profile/main-info/MainInfo.constants'
+import { styles } from '~/containers/tutor-profile/main-info/MainInfo.styles'
+import { snackbarVariants, myProfilePath } from '~/constants'
+
+const MainInfo = () => {
```
Comment (maarekua) [inline src/containers/tutor-profile/main-info/MainInfo.jsx:25]: Naming

Comment (Tolik170) [reply src/containers/tutor-profile/main-info/MainInfo.jsx]: fixed

**Review by romanortynskyi — APPROVED** [review]

**Pull request merged**